config/routes.rb

```diff
@@ -1,10 +1,14 @@
 Rails.application.routes.draw do
   devise_for :users
+
   root to: 'pages#home'
-  resources :users, only: %i[show edit update destroy] do
+
+  resources :users, only: %i[show] do
     resources :boats, only: %i[index show new create edit update]
```

Comment:
I took a look and I don't think the boats should be nested under users. The routes as they are right now don't work, you cannot make a booking or do the controllers (because they take info from the params which are set in the routes). We should discuss this evening.

Reply:
Totally, we should check our routes. I'll take a look now before the class starts and show you guys what I did.